### ISSUE TITLE
hosting request from scientists wanting new URL....wim.usgs.gov/dashboard/......

### ISSUE THREAD

**Issue by jamvelk** [USGS-WiM/Geonarratives#6]
I have a hosting request from a scientist that would like to host an AGOL dashboard with WIM, however would like the URL to be:

wim.usgs.gov/dashboard/......and then name of his dashboard.

This is the first dashboard we will host, and I believe we hosted one app before, but just hosted that under /geonarrative.

What are the options for hosting if someone wants a different URL?

I spoke to Mitch about this already and he mentioned we could likely do wim.usgs.gov/geonarrative/dashboard,  but if it’s right off the WIM subdomain (like wim.usgs.gov/dashboard) some may be reserved or restricted. 

The most practical option I can think of is if we host future webapps (maps, dashboards, ect) that are not geonarratives would be wim.usgs.gov/app/...nameofapp.

So just wanted to run it by you both @HansVraga and @aaronstephenson, what I should tell the scientist about his request?

Thanks!

**Comment by HansVraga** [comment]
Jamie,
The Dashboard review process is following the Geonarrative requirements, correct? I'm wondering if we should keep that Geonarrative path - do they have reasons for wanting to move away from it?


Hans Vraga | he/him<https://www.mypronouns.org/>

Program Coordinator (acting) / Project Manager

Web Informatics and Mapping (WIM)<https://wim.usgs.gov/>
Upper Midwest Water Science Center
________________________________
From: jamvelk ***@***.***>
Sent: Thursday, September 30, 2021 10:44
To: USGS-WiM/Geonarratives ***@***.***>
Cc: Vraga, Hans W ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] [USGS-WiM/Geonarratives] hosting request from scientists wanting new URL....wim.usgs.gov/dashboard/...... (#6)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



I have a hosting request from a scientist that would like to host an AGOL dashboard with WIM, however would like the URL to be:

wim.usgs.gov/dashboard/......and then name of his dashboard.

This is the first dashboard we will host, and I believe we hosted one app before, but just hosted that under /geonarrative.

What are the options for hosting if someone wants a different URL?

I spoke to Mitch about this already and he mentioned we could likely do wim.usgs.gov/geonarrative/dashboard, but if it’s right off the WIM subdomain (like wim.usgs.gov/dashboard) some may be reserved or restricted.

The most practical option I can think of is if we host future webapps (maps, dashboards, ect) that are not geonarratives would be wim.usgs.gov/app/...nameofapp.

So just wanted to run it by you both @HansVraga<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FHansVraga&data=04%7C01%7Chvraga%40usgs.gov%7C4fd9905055174cc2daec08d98429d420%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637686137452422465%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=CCRyTk7CACy%2FLjhqpYKxJCUCR6xnkAWA9z8Ji%2F2GMgU%3D&reserved=0> and @aaronstephenson<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Faaronstephenson&data=04%7C01%7Chvraga%40usgs.gov%7C4fd9905055174cc2daec08d98429d420%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637686137452432418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=5TZoXcKZLZCnG59ek6owkOEU%2B33NWgEE0KeBA2FjkFc%3D&reserved=0>, what I should tell the scientist about his request?

Thanks!

**Comment by jamvelk** [comment]
@HansVraga that's correct. I think they just wanted it to not say "geonarrative" since technically it's not, but just a dashboard. It is still an AGOL product and follows the same review process I believe.
 
I told them I'd run it by the WIM Admins, so that's why I'm asking you :)

I do like giving the option of adding a sub-folder of wim.usgs.gov/geonarrative/dashboard/nameofstorymap, if they want.  

What do you think?

**Comment by jamvelk** [comment]
hi @HansVraga , just wanted to follow up on this, if I want to give the scientist a definitive answer, should I just tell them that we are stinking with the hosting as is, and not pursuing other URL options at this time? (mind if I copy you on the email if/when I do?)
